Flow.Launcher/MainWindow.xaml

```diff
@@ -193,6 +193,12 @@
                             </ContextMenu>
                         </TextBox.ContextMenu>
                     </TextBox>
+
+                    <StackPanel x:Name="ClockPanel" Style="{DynamicResource ClockPanel}" IsHitTestVisible="False">
```

Comment:
what's ishittestvisible?

Reply:
it make the item non-clickable. in this case, focus the textbox when click clock item.

Reply:
that's cool!